app/views/shared/_gfw_assets.html.slim

```diff
@@ -0,0 +1,4 @@
+- if Rails.env.production?
+  script src="http://gfw-assets.s3.amazonaws.com/static/gfw-assets.latest.js" id="loader-gfw" data-current=".shape-climate"
+- else
+  script src="http://gfw-assets.s3.amazonaws.com/static/gfw-assets.nightly.js" id="loader-gfw" data-current=".shape-climate"
```

Comment:
I prefer something like this:
`assets_version= Rails.env.production? 'latest' : 'nightly'`
but better is:
in dot env file add `ASSETS_VERSION=latest` and use this like `${ENV['ASSETS_VERSION']}`